ocs_ci/templates/openshift-infra/monitoring/configuring_pvc.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: v1
+kind: ConfigMap
```

Comment:
@Akarsha-rai @vasukulkarni @petr-balogh would it make sense to create a class for monitoring to be inherited by ConfigMap class?